.storybook/index.scss

```
@@ -1,9 +1,11 @@
 ////
 /// Entry point.
 /// @group storybook
-/// @copyright IBM Security 2019 - 2020
+/// @copyright IBM Security 2019 - 2021
 ////
 
+@import '@carbon/themes/scss/themes';
+@import '@carbon/themes/scss/tokens';
```

With the suggestion below, only the Carbon theme definition map should be required:

```suggestion
@import '@carbon/themes/scss/theme-maps';
```